recipes/mo-catnip/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mo-catnip" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 491e72214f2018dcd57e580956c97eb91b6df1cd6172b85a7e9e0da985733e54
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This package looks like it could be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
I have changes it to noarch: python